src/lib/utitlities/fetch-raw-string.ts

```diff
@@ -0,0 +1,4 @@
+export async function fetchRawString(url: string) {
+  const response = await fetch(url)
+  return await response.text()
```

Comment:
Another thing to carefully consider: this requires network calls, which is... not a good strategy. Should look at Vite's `import.meta.glob`.

https://vitejs.dev/guide/features#glob-import

Reply:
Implemented in https://github.com/thanhnguyen2187/crypta/pull/28/commits/41f394ca0abe765c6442bedc36323a2e4fc8db22.